osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs

```diff
@@ -14,12 +14,14 @@ namespace osu.Game.Rulesets.Osu.Objects
     /// </summary>
     public class SliderTailCircle : SliderEndCircle
     {
+        public bool JudgeAsSliderTick = false;
```

Comment:
Looks like this is a vestige of the old boolean-flag-based implementation and should not be here anymore. Nothing ever sets this.